README.md

```diff
@@ -1,65 +1,130 @@
-FoxBox
-======
+# FoxBox
 
 [![Build Status](https://travis-ci.org/fxbox/foxbox.svg?branch=master)](https://travis-ci.org/fxbox/foxbox)
 [![License](https://img.shields.io/badge/license-MPL2-blue.svg)](https://raw.githubusercontent.com/fxbox/foxbox/master/LICENSE)
 
 
-## Target
+## Technologies
 
-Target hardware for prototyping is the Raspberry Pi 2. It's ARMv7 with Neon, but building without Neon support should do for now. Target OS is the latest Raspbian which is based on Debian 8.0 Jessie.
+### Rust
 
+We're using Rust for the daemon/server. Currently v1.8.x nightly is required. As
+of 2016-02-04:
 
-## Toolchain
+```bash
+$ rustc -V
+rustc 1.8.0-nightly (dea183aa8 2016-02-02)
```

Comment:
That's not the version to use anymore. Current known good rustc revision is fae516277b6da46b6c1cf568765c90fad2f9ae4b